src/components/landing-page/call-to-action/CallToAction.jsx

```
@@ -1,22 +1,122 @@
+          </div>
+        </div>
+      </div>
+      <div className="flex flex-col items-center justify-center w-full p-[3%] gap-4 bg-[#374151] h-auto lg:h-[912px]">
+        <p className="lg:w-[496px] text-3xl lg:text-5xl mt-[32px] font-bold text-white text-center ">
+          Why Choose AURORA?
+        </p>
+        <p className="text-center font-normal text-[#D1D5DB] text-xl">
+          Our AI-powered platform offers unique advantages for language learners
+        </p>
+        <div className="gap-4 grid md:grid-cols-2  lg:grid-cols-3   justify-center items-center">
+          {whyChooseAuruora.map((content, i) => {
```

ESLint

[error] 'whyChooseAuruora' is not defined.

(no-undef)